projects/public-search/src/app/scss/styles.scss

```diff
@@ -24,3 +24,17 @@
 .toast-container .ngx-toastr {
   width: 450px !important; // override width for all toastr message
 }
+.custom-select {
+  border: none;
+  background-color: transparent;
+  color: white;
+}
+select#sort-criteria.custom-select {
+  background-color:#343a40;
```

Comment:
is it possible to use bootstrap variable here ? maybe `$table-dark-color` ?

Reply:
The changes in this file are to overwrite the style of the sort-list ng-core widget. 
I'm not sure how to use bootstrap in this case.

Reply:
@vgranata

You can check https://github.com/rero/rero-ils-ui/blob/staging/projects/admin/src/app/acquisition/acquisition.scss (and some other files)
```scss
@import '~bootstrap/scss/_functions';
@import '~bootstrap/scss/_variables';
@import '/projects/admin/src/app/scss/variables';
```
allows to use bootstrap variables, bootstrap functions and project variables.
```scss
margin-bottom: map-get($spacers, 1);
```
you can use 1 (small gap space) to 5 (big gap space)
If you can't use a bootstrap variable for the background color ; define a project variable into `variables.scss` and use this variable in this file